src/Sparrow/Json/BlittableWriter.cs

```diff
@@ -286,31 +314,28 @@ public static void CleanPropertyArrayOffset()
             _intBuffer = null;
         }
 
+#if NET6_0_OR_GREATER
```

Comment:
Please check the above warning about:
`BlittableWriter<TWriter>._propertyArrayOffset`